### ISSUE TITLE
records-ui: Display record creation date (not just publication data metadata)

### ISSUE THREAD

**Issue by krzysztof** [zenodo/zenodo#1430]
It might be relevant to see when the record first appeared on Zenodo (upload date), besides the publication date metadata.

**Comment by michaip4** [comment]
Indeed, that would be a great feature. With display of the date/time when a preprint is submitted, Zenodo could be used to establish scientific priority, exactly like arXiv. That would be a great additional functionality.
